org.flatpak.am-okay.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+
+  <id>org.flatpak.am-okay</id>
+  
+  <name>am-okay</name>
+  <summary>The `am-okay` program allows dynamically to copy/move files/directories (via the terminal)</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  
+  <description>
+    <p>
+      The <code>am-okay</code> program allows to copy or move files or directories in a dynamic mode via a console (terminal).
+    </p>
+    <p>
+      With the <code>am-okay</code> program, you can dynamically copy or move files or directories where access is permitted. The program itself partly imitates the Linux <code>cp</code> , <code>mv</code> commands but dynamically. You can tag files or  directories as items to copy or move via a console (terminal), and paste these items via another  console (terminal).
+    </p>
+    <p>
+      A lot of actions are possible, you can even tag files or directories in arrays and paste them  wherever you want in your Linux system.
+    </p>
+  </description>
+  
+  <icon type="stock">org.flatpak.am-okay</icon>
+  
+  <categories>
+    <category>Utility</category>
+    <category>FileTools</category>
+  </categories>
+  
+  <provides>
+    <binary>am-okay</binary>
+  </provides>
+
+  <developer id="com.baldeuniversel">
+    <name>Baldé Amadou</name>
+    <name xml:lang="en">Baldé Amadou</name>
+  </developer>
+
+  <releases>
+    <release version="3.1.2" date="2024-03-05" />
+  </releases>
+
+  <keywords>
+    <keyword translate="no">utility</keyword>
+    <keyword translate="no">am-okay</keyword>
+    <keyword translate="no">ubuntu</keyword>
+    <keyword translate="no">linux</keyword>
+    <keyword translate="no">terminal</keyword>
+  </keywords>
+
+  <url type="homepage">https://github.com/baldeuniversel/am-okay</url>  
+  <url type="vcs-browser">https://github.com/baldeuniversel/am-okay.git</url> 
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/baldeuniversel/am-okay/blob/main/am-okay/usr/local/share/screenshot/screenshot_scenario_1_am-okay.png</image>
+      <caption>Use case 1, `am-okay` scenario</caption>
+    </screenshot>
+
+    <screenshot type="default">
```

Comment:
There can only be one `default`-type image.

```suggestion
    <screenshot>
```

Reply:
Done !